public/style.css

```diff
@@ -1,51 +1,132 @@
-.sidebar {
-    position: fixed;
-    top: 0;
-    bottom: 0;
-    left: 0;
-    z-index: 100;
-    padding: 90px 0 0;
-    box-shadow: inset -1px 0 0 rgba(0, 0, 0, .1);
-    z-index: 99;
+
+
+:root, [data-bs-theme=light] {
+  --bs-breadcrumb-margin-bottom: 0.2rem;
 }
 
-@media (max-width: 767.98px) {
-    .sidebar {
-        top: 11.5rem;
-        padding: 0;
+/* Defaults. */
+/* for extra small screens, show a similar sidebar but full width and only if the menu button is pressed. */
+@media(max-width: 768px) {
+    button.navbar-toggler {
+        display: inline;
+        border-radius: 0;
+        border: 0;
+        background-color: var(--bs-secondary-bg);
+    }
+    nav#sidebar {
+        display: none;
+    }
+    nav#sidebar a.nav-link-extra { margin-top: 1em };
+    nav#sidebar a.nav-link {
+        display: block;
+        width: 100%;
+        font-size: 150%;
+        height: 40px;
+    }
+    nav#sidebar a.nav-link:click {
+        background-color: var(--bs-primary-bg);
+    }
+
+    nav#sidebar.show {
+        position: fixed;
+        display: flex;
+        flex-wrap: wrap;
+        align-content: start;
+        height: 100vH;
+        width: 220px;
```

Comment:
the menu should be full screen not partial screen.

Reply:
I agree. But then the options would be at the left side, far away from the (right side) menu button.
I thought this was more ergnonomic.
How about moving the menu button to the left? That would solve both issues.

Reply:
Since English is a left to right language that would make sense as that is where your eyes start reading